org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/DoEntity.java

```diff
@@ -76,15 +76,19 @@ public void putNode(String attributeName, DoNode<?> attribute) {
   }
 
   /**
-   * Adds new value to attribute map. The value is wrapped within a {@link DoValue}.
+   * Associates the specified value with the specified attribute name in this entity. If the entity previously contained
+   * a mapping for the attribute name, the old value is replaced by the specified value.<br>
+   * The value is wrapped within a {@link DoValue}.
    */
   @Override
   public void put(String attributeName, Object value) {
     doValue(attributeName).set(value);
   }
 
   /**
-   * Adds new list value to attribute map. The value is wrapped within a {@link DoList}.
+   * Associates the specified list value with the specified attribute name in this entity. If the entity previously
+   * contained a mapping for the attribute name, the old list value is replaced by the specified value.<br>
+   * The value is wrapped within a {@link DoList}.
    */
   @Override
   public <V> void putList(String attributeName, List<V> value) {
```

Comment:
Does this method still create a new empty list which will be immediately thrown away?

Reply:
No - the javadoc comment is misleading. I'll change it.